opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/ServerTracingAutoConfiguration.java

```diff
@@ -34,18 +37,32 @@
 public class ServerTracingAutoConfiguration {
     private static final Log log = LogFactory.getLog(ServerTracingAutoConfiguration.class);
 
+    private final ObjectProvider<List<ServletFilterSpanDecorator>> servletFilterSpanDecorator;
+    private final ObjectProvider<List<HandlerInterceptorSpanDecorator>> interceptorSpanDecorator;
+
+    public ServerTracingAutoConfiguration(ObjectProvider<List<ServletFilterSpanDecorator>> servletFilterSpanDecorator,
+                                          ObjectProvider<List<HandlerInterceptorSpanDecorator>> interceptorSpanDecorator) {
+        this.servletFilterSpanDecorator = servletFilterSpanDecorator;
+        this.interceptorSpanDecorator = interceptorSpanDecorator;
+    }
+
     @Bean
     @ConditionalOnMissingBean(TracingFilter.class)
     public FilterRegistrationBean tracingFilter(Tracer tracer, WebTracingProperties tracingConfiguration) {
         log.info("Creating " + FilterRegistrationBean.class.getSimpleName() + " bean with " +
```

Comment:
Since not using slf4j, we should wrap that in `log.isInfoEnabled()` since we are processing some stuff to log

Reply:
I guess that is fine as this is invoked only at app start time.

`String.format() ` should help with readability.

Reply:
Maybe but a bad practice especialy in lib, Spring wrap error in isErrorEnabled() if message need to be processing!